### ISSUE TITLE
Dark mode misses some elements

### ISSUE THREAD

**Issue by Alch-Emi** [etesync/etesync-web#260]
Using the production etesync-web from pim.etesync.com on Firefox 113.0.1, my calendar view looks like this
![image](https://github.com/etesync/etesync-web/assets/38897201/b93b593e-0357-4056-a516-ad37c7b97595)

A few things that might need to be changed:
- The text of the today/back/next buttons have next-to-zero contrast with the button background.  Seems like they are using the original button text color.
- Hovering the today/back/next buttons changes the color drastically.  I believe this to also be the use of a light-mode color
- "inactive" days of the month (eg Monday the 31st in this picture) use a startlingly bright color
- The hamburger menu at the top right and the page title use very different colors
- The white text in the yellow header has a very low contrast

A couple of these things might just be design decisions that I disagree with, but some are severe usability and accessibility barriers which should really be addressed, like the near-invisible text of the buttons.  I'm not sure if this is visible for everyone or just me, but it kinda makes dark mode a bit unusable.

**Comment by virusperfect** [comment]
Yeah I'm having the same issue. Dark mode is pretty much unusable.
